.circleci/config.yml

```diff
@@ -38,17 +32,56 @@ commands:
 
   sync:
     parameters:
-      from:
-        type: string
-      to:
+      dist_path:
         type: string
-      overwrite:
-        default: false
-        type: boolean
     steps:
       - run:
-          name: Deploy to S3
-          command: "aws s3 sync << parameters.from >> << parameters.to >><<# parameters.overwrite >> --delete<</ parameters.overwrite >>"
+          name: Set default environment variables
+          command: |
+            cd && touch $BASH_ENV
+
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+              echo 'export INFRA_PATH=iac/conf/websites/prod/${CIRCLE_PROJECT_REPONAME}' >> $BASH_ENV
+              echo 'export DEPLOYMENT_ENV=${CIRCLE_PROJECT_REPONAME}' >> $BASH_ENV
+              echo 'export GIT_BRANCH=${CIRCLE_BRANCH}' >> $BASH_ENV
+            elif [ "${CIRCLE_BRANCH}" == "develop" ]; then
+              echo 'export INFRA_PATH=iac/conf/websites/dev/${CIRCLE_PROJECT_REPONAME}-staging.interblock.io' >> $BASH_ENV
+              echo 'export DEPLOYMENT_ENV=${CIRCLE_PROJECT_REPONAME}-staging.interblock.io' >> $BASH_ENV
+              echo 'export GIT_BRANCH=${CIRCLE_BRANCH}' >> $BASH_ENV
+            else
```

Comment:
the last else condition is temporary right?

Reply:
It can be if you want. Right now this setup will give you one of three outcomes:
master branch => voyager prod
develop branch => voyager-staging
any other branch => voyager-feature

I did this so people can see their feature branch work deployed to aws. 

https://voyager-feature.interblock.io will display the latest feature branch build

Reply:
Woudl be nice, but I think we will end up overriding each other very often, we all push several time per day, maybe on this we can ask what the others think

Reply:
Now that I think about it I was using the circleci artifacts to share features, so we can remove it without loosing this capability:
https://15583-99653950-gh.circle-artifacts.com/0/tmp/voyager/app/dist/index.html

Reply:
Removed the feature branch deployment.